dspace/config/launcher.xml

```diff
@@ -373,4 +373,11 @@
             <class>org.dspace.app.util.Version</class>
         </step>
     </command>
+    <command>
+        <name>initialize-entities</name>
```

Comment:
since Entities name will be discontinued. I think this task could be renamed to items, or itemtypes? I don't know if there will be sub-tasks. Perhaps could have a generic name, like others tasks have (example: oai, ...)